### ISSUE TITLE
Robie.blend file

### ISSUE THREAD

**Issue by Yzubi** [bladecoder/bladecoder-adventure-tests#1]
The Robie 3D model has been made in Blender, right? Can you upload the .blend file to the Github repo for reference?

**Comment by bladecoder** [comment]
Of course. I uploaded it in the 'adventure-test-project/raw' folder: robie.blend.

This is the model used in the 3d test where it is mixed the 3d character (Robie) with the 2d background.

Best Regards!
